spec/integrations/workflow_run_spec.rb

```diff
@@ -79,6 +79,12 @@
 
         expect(file.size).to be > 0
       end
+
+      it 'downloads evidence folder' do
+        file = onfido_api.download_evidence_folder(workflow_run_id)
+
+        expect(file.size).to be > 0
```

Comment:
could it also check whether it's a (temp)file?

Reply:
updated the test to extract the zip content instead 👍